Source/IntegrationTests/LitTests.cs

```diff
@@ -239,7 +239,7 @@ public class LitTests {
               return false;
             }
 
-            if (arguments.Any(arg => arg is "/compile:3" or "/compile:4" or "run" or "translate")) {
+            if (arguments.Any(arg => arg is "/compile:3" or "/compile:4" or "run")) {
```

Comment:
This doesn't seem right to me, I added "translate" because there were important backend-specific tests that weren't being run for all backends before that.

Reply:
How can you test `dafny translate` without back-end specific code? Testing `dafny translate` implies to me that you want to be back-end specific, which is also true for the tests in this PR.

Reply:
No I couldn't convert them to `%testDafnyForEachCompiler` as you say, but I did manually add missing backend cases. https://github.com/dafny-lang/dafny/blob/master/Test/comp/separate-compilation/usesTimesTwo.dfy for example.

The name `NeedsConverting()` is a bit out of date, since the idea is just to identify tests that use backends but don't test all backends.

Reply:
What's the use of adding `// NONUNIFORM: using dafny translate` to all tests that exercise `dafny translate` ?

Reply:
I thought `NONUNIFORM` meant you tests some back-ends but not others, but now you're suggesting it will mean that you're not using `%testDafnyForEach...`

Reply:
It's just so that we can make the conscious choice to either switch to `%testDafnyForEachCompiler` (ideally) or document that the test can't be expressed in a way that automatically works across all backends. Adding `// NONUNIFORM` means the scan won't complain about the test any more, but we can periodically audit the non-uniform tests and at least manually add new backends, or find ways of making them uniform.

Reply:
> It's just so that we can make the conscious choice to either switch to %testDafnyForEachCompiler (ideally) or

For `translate` you can't switch to that so I don't see a choice.

> but we can periodically audit the non-uniform tests and at least manually add new backends, or find ways of making them uniform.

You can do that by looking for `dafny translate` !

Reply:
> You can do that by looking for dafny translate !

The idea was to have `// NONUNIFORM` on all tests that don't automatically pick up new compilers, regardless of why. We could modify https://github.com/dafny-lang/dafny/blob/master/.github/workflows/msbuild.yml#L59-L60 to look for `dafny translate` as well if you want.

Reply:
> The idea was to have // NONUNIFORM on all tests that don't automatically pick up new compilers, regardless of why

What's the use? The reason you gave was that so you can "periodically audit the non-uniform tests and at least manually add new backends", but you don't need this semantic comment for that. Any test that specifies a target, either `--target`, or using `translate`, is non-uniform.

What I think is useful, is to complain about usages of `dafny run` or `dafny build` that don't use `--target`, because they are uniform and could be tested against all back-ends, so it's wasted tested opportunity not to do so.

Reply:
btw, do you know why a file like this one https://github.com/dafny-lang/dafny/blob/master/Test/c%2B%2B/extern.dfy#L1 isn't flagged by the check?

Reply:
I don't offhand, I'll look into it separately from this PR. And that means for now I won't object to this change. :)

Reply:
After offline conversation I realize this is a good change - // NONUNIFORM should only be used for tests of backend-specific behaviour, like ExternJavaString.dfy. as part of my action item I'll remove the tag from tests like that.